server/server.go

```diff
@@ -64,6 +65,9 @@ type Server struct {
 	// transaction pool
 	txpool *txpool.TxPool
 
+	//	event tracker
+	tracker *tracker.Tracker
+
```

Comment:
Server doesn't have to have tracker because server will not use it directly

Reply:
[Resolved](https://github.com/0xPolygon/polygon-edge/pull/386/commits/585d357ff9ff834ac33cc2fd98a060539046f5dc)